src/app/settings/action-triggers/components/action-triggers-list/action-triggers-list.component.scss

```diff
@@ -0,0 +1,65 @@
+.c-action-triggers-list {
+  width: 100%;
+  &__search {
+    margin-bottom: 1.5rem;
+    display: flex;
+    align-items: center;
+    & input {
+      width: 100%;
+      padding: 0.5rem;
+    }
+    & sofie-dropdown-button {
+      margin-left: 1rem;
+    }
+  }
+  &__items {
+    max-height: calc(100vh - 233px);
```

Comment:
Yes, it's easy way to make window height 100% when we have static content above. There is an option to make the whole window 100% or 100vh height and occupy parts of the window relative (10%) or with exact measurements (10vh or 100px), but this will require significantly more effort and is done when we have relative information on top (or a responsive page)

Reply:
I don't think it is the right approach to have a subcomponent  base its size on the window size and another component outside of its node tree. If we change the header size, then this computation will no longer be valid.
We try to have the components take up the place that they are given on the page by their parent components.

Reply:
The problem we have here and I'm trying to solve with `calc(100vh - 233px)` is:
- We want to take the space witch the parent component gives us, also want to set internal scrolling to some of the components (as action triggers list. and actions list). In addition, we don't want to work with exact units (`px`, `rem `or `em`) of measurement because for me it looks good on a (1920x1080px) with a max-height of 700px, but on another larger resolution it won't look good (the lists will be smaller then the screen), and on a small screens we will have a second scroll. I'm not sure how else we can accomplish the designs. Maybe we should change the design approach to be more elementary (not with internal scrolls). We can leave it for discussion after we go through the design meeting next week
I'm open to suggestions if anyone has an idea how to do the design without using max-height with `vh `or `px`

Reply:
As I see, the issue is that we have a branch in the DOM tree, where not every component (and element) is adhering to the guideline of using the space given to it. If the parent elements fill out the space available, then we won't need this. One approach is to use flex (or grid) to arrange the structural components of the page.